apps/studio/prisma/schema.prisma

```diff
@@ -109,11 +109,10 @@ enum ResourceType {
 }
 
 model User {
-  id            String  @id @default(cuid())
-  name          String
-  email         String  @unique
-  phone         String
-  preferredName String?
+  id    String  @id @default(cuid())
+  name  String
+  email String  @unique
+  phone String?
```

Comment:
actually I thought of keeping the preferred name for future use cases like making the UI more friendly e.g. Hi John, ...